### DISCUSSION TITLE
Nice idea, Permissions, & Service Logins

### DISCUSSION THREAD

**Discussion by Debranis** [lavalink-devs/lavaplayer#138] [Q&A] [answered]
I saw this on the Simple Voice Chat Discord server and was intrigued.  Nice idea and I am excited to use it on my server.  Thank you for building it.

I do have two questions though.

First, are there any plans to incorporate permission nodes to control some of the following:

- Who can utilize the mod on the server
- Who can utilize player controls (both their own streams and the streams of others)

Second, are there any plans or does it already support the ability to incorporate service login to access playlists, favorites, etc.

- I want to access my own YouTube Music and access playlists that I have created (or Twitch, etc.) and have it play in a Voice Chat group of my making.
- I want to allow others to login to their accounts (YouTube, Twitch, etc.) and access their own playlists and have it play in a Voice Chat group that they create.

Thank you again,

Debranis

**Comment by SerenModz21** [comment, marked as answer]
hi @Debranis! i would recommend you copy the above and send it to https://github.com/ItzDerock/simplevoicechat-music, which is the correct github repo for the voice chat addon. lavaplayer is simply a library that the addon uses behind the scenes. 

note: i don't have any affiliation with the addon or lavaplayer. i am just another voice chat user like yourself.

**Reply by Debranis** [reply]
My apologies, thank you for the clarification/correction

**Comment by devoxin** [comment]
Not relevant to this repository. As Seren has stated, please refer to the mod repository for things like this.
